Lombiq.VueJs/package.json

```diff
@@ -14,23 +12,31 @@
   },
   "devDependencies": {
     "@vue/compiler-sfc": "^3.2.29",
+    "eslint": "^8.17.0",
     "fancy-log": "^2.0.0",
     "fs": "0.0.1-security",
-    "gulp": "4.0.2",
-    "gulp-all": "1.1.0",
-    "gulp-clean-css": "4.3.0",
-    "gulp-plumber": "1.2.1",
-    "gulp-rename": "2.0.0",
-    "gulp-rollup": "2.17.0",
-    "gulp-sass": "5.1.0",
-    "gulp-uglify": "3.0.2",
-    "rollup-plugin-alias": "1.5.1",
-    "rollup-plugin-buble": "0.19.6",
-    "rollup-plugin-commonjs": "8.4.1",
+    "glob": "8.0.3",
+    "rollup": "2.79.1",
+    "@rollup/plugin-alias": "3.1.9",
+    "@rollup/plugin-commonjs": "22.0.2",
     "rollup-plugin-eslint": "7.0.0",
-    "rollup-plugin-json": "4.0.0",
-    "rollup-plugin-node-resolve": "4.1.0",
-    "rollup-plugin-replace": "2.2.0",
-    "del": "6.0.0"
+    "@rollup/plugin-json": "4.1.0",
+    "@rollup/plugin-node-resolve": "14.1.0",
+    "@rollup/plugin-replace": "4.0.0",
+    "terser": "5.15.0",
+    "del": "6.0.0",
+    "npm-run-all": "^4.1.5"
+  },
+  "scripts": {
+    "build": "run-p --print-name --continue-on-error \"build:*\"",
+    "compile": "run-p --print-name --continue-on-error \"build:*\"",
+    "clean": "run-p --print-name --continue-on-error \"clean:*\"",
+    "watch": "true",
```

Comment:
I assume you kept this until the end - or is something keeping us from monitoring the vue apps/sfcs and rerun the compiler pipelines for them?

Reply:
I'm not aware of anything implemented for it. Wasn't in the GE version either. Open a followup issue?

Reply:
If it wasn't in GE either, then yes, a follow-up issue will be best.

Reply:
Please link the issue here.